app/views/feeds/_form_main.html.erb

```diff
@@ -0,0 +1,203 @@
+    <%= render "form_auth_tokens", feed: feed, form: form %>
+  </div>
+
+  <div class="col-12">
+    <hr>
+    <h3><%= t(".headers.ad_zones") %></h3>
+    <p><%= t(".help.ad_zones") %></p>
+
+    <%= form.check_box :billboard, value: 0 %>
+    <%= form.label :billboard, class: "me-2" %>
+
+    <%= form.check_box :house, value: 0 %>
+    <%= form.label :house, class: "me-2" %>
+
+    <%= form.check_box :paid, value: 0 %>
+    <%= form.label :paid, class: "me-2" %>
+
+    <%= form.check_box :sonic_id, value: 0 %>
+    <%= form.label :sonic_id, class: "me-2" %>
+
+  </div>
+  <div class="col-12">
+    <hr>
+  </div>
+<% end %>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :url %>
+    <%= form.label :url %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.url") %>"><span class="material-icons">help</span></button>
+</div>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :new_feed_url %>
+    <%= form.label :new_feed_url %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.new_feed_url") %>"><span class="material-icons">help</span></button>
+</div>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :enclosure_prefix %>
+    <%= form.label :enclosure_prefix %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.enclosure_prefix") %>"><span class="material-icons">help</span></button>
+</div>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.number_field :display_episodes_count %>
+    <%= form.label :display_episodes_count %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.display_episodes_count") %>"><span class="material-icons">help</span></button>
+</div>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.number_field :display_full_episodes_count %>
+    <%= form.label :display_full_episodes_count %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.display_full_episodes_count") %>" data-bs-html="true"><span class="material-icons">help</span></button>
+</div>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.select :episode_offset_seconds, episode_offset_options %>
+    <%= form.label :episode_offset_seconds %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.episode_offset_seconds") %>"><span class="material-icons">help</span></button>
+</div>
+
+<div class="col-12 mb-4">
+  <hr>
+  <h3><%= t(".headers.audio_format") %></h3>
+  <p><%= t(".help.audio_format") %></p>
```

Comment:
TODO: like the Ad Zones checkboxes, it felt complicated to break the `audio_format` attribute into each of these selects in this PR. can address as fallout.